server/src/services/food.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose, { ObjectId } from "mongoose";
+import { FoodAttributes } from "../models/Food";
+import { FoodModel } from "../models/Food";
+
+// Create a single new food item
+async function create(food: FoodAttributes): Promise<FoodAttributes> {
+  return FoodModel.create(food);
+}
+
+// Creates many food items
+async function createMany(foods: FoodAttributes[]): Promise<FoodAttributes[]> {
+  return FoodModel.insertMany(foods);
+}
+
+// Delete a single food item by an id
+async function deleteFoodById(foodId: ObjectId): Promise<void> {
+  FoodModel.deleteOne(foodId);
```

Comment:
There's a chance that the food doesn't exist
Let's return the food that got deleted, or null if it doesn't exist